src/main/java/org/apache/accumulo/testing/TestEnv.java

```diff
@@ -97,6 +97,13 @@ public Configuration getHadoopConfiguration() {
       hadoopConfig.set("fs.file.impl", org.apache.hadoop.fs.LocalFileSystem.class.getName());
```

Comment:
Unrelated to your change, but this seems really odd.

Reply:
This is the implementation to handle HDFS filesystems that are `file://` URI-based instead of `hdfs://`. It's only ever used if the configured DFS filesystem is local.

Reply:
Thanks @ctubbsii .  I knew that but had forgotten.